Configurations/10-main.conf

```diff
@@ -934,28 +934,28 @@ sub vms_info {
     # in order to build "universal" binary and allow OpenSSL take
     # advantage of NEON when it's available.
     #
-    "android-armeabi" => {
-        inherit_from     => [ "android", asm("armv4_asm") ],
+    "android-armeabi-base" => {
+        inherit_from     => [ asm("armv4_asm") ],
```

Comment:
> Move asm("mips32_asm") to compiler-specific targets sounds strange.

Not to seasoned MIPS programmer :-) Normally, i.e. in cases other than Android, MIPS systems support multiple ABIs. And in such cases you'd actually "tie" perlasm_scheme and compiler flags together, i.e. the have to match. But my objection is really more about minimizing templates. It does make code more readable. I mean when you look at say android-mips, you have to unravel one less template. And it's also better for sheer amount of lines. I mean pair of perlasm_scheme in android-mips and android-mips-clang *is* shorter than template...